### ISSUE TITLE
Ghost start fails on reboot when ghost is installed in a digitalocean block storage

### ISSUE THREAD

**Issue by PhoenixPeca** [TryGhost/Ghost-CLI#390]
Can somebody please tell me why this is happening in my Ubuntu 16.04.2 droplet?

![image](https://user-images.githubusercontent.com/9730242/28723598-d6017a9e-73e9-11e7-9dd2-1a5fe92a1909.png)

On `ghost intall` it works perfectly, but if I restart the server this error shows.

`ghost run` is working well though.

Is this a bug or something?

As for the log:
````
Debug Information:
    Node Version: v6.11.1
    Ghost-CLI Version: 1.0.0
    Environment: production
    Command: 'ghost start'
Error occurred running command: '/bin/sh -c sudo systemctl start ghost_www-spud-cf'

Exit code: 1
````

**Comment by ErisDS** [comment]
Did you run `ghost install` without flags, and did you choose "yes" to creating a systemd setup?

**Comment by PhoenixPeca** [comment]
hi @ErisDS,
I have noticed that this error only occurs when ghost is intalled in a block storage in a digitalocean droplet. maybe because the systemd daemon has been loaded before the system can even mount the block storage on boot, such that the linked file is broken. so as a workaround I have added the following line in the crontab file of root:

`@reboot systemd daemon-reload`

I hope this helps.
